src/generated/composite.rs

```diff
@@ -24,14 +24,8 @@ use crate::errors::{ConnectionError, ParseError};
 use crate::x11_utils::GenericEvent;
 #[allow(unused_imports)]
 use crate::x11_utils::GenericError;
-#[allow(unused_imports)]
-use super::xproto;
-#[allow(unused_imports)]
-use super::render;
-#[allow(unused_imports)]
-use super::shape;
-#[allow(unused_imports)]
 use super::xfixes;
+use super::xproto;
```

Comment:
I tried bringing this change to the current code generator. This led to another bug found in xcbgen: https://gitlab.freedesktop.org/xorg/proto/xcbproto/-/issues/15